include/termcolor/termcolor.hpp

```diff
@@ -393,7 +521,7 @@ namespace termcolor
             stream << "\033[40m";
         #elif defined(TERMCOLOR_OS_WINDOWS)
             _internal::win_change_attributes(stream, -1,
-                0   // grey (black)
+                0 | FOREGROUND_INTENSITY   // grey (black)
```

Comment:
Could you please explain this change?

Reply:
Whoops, I think this was trying to correct the high color for black.

My local branch seems to have this corrected (only part of the diff that I hadn't uploaded to the PR for some reason!)

Reply:
Should be fixed now!